src/components/BuildBaronAndAnnotations/BBFIleTicket.tsx

```diff
@@ -23,7 +23,6 @@ export const BBFileTicket: React.FC<BBFileTicketProps> = ({
   createdTicketsCount,
 }) => (
   <>
-    <TicketsTitle margin>Create a new ticket in Jira </TicketsTitle>
```

Comment:
I'm a bit concerned about the design choice of removing this title. If there are already filed tickets, then the user will see a title, a list of tickets, and a "file another ticket" button making the use case clear. However, if there are no tickets yet, all the user will see is the button with no explanation. I think we need another design or screenshot with just the button and no filed tickets to make sure that things are still clear.

Reply:
Fair--I removed this during earlier talks with Lara! I'm going to switch the title to what was in the most recent design